src/pages/search/registration_search/filters/AdvancedSearchRow.tsx

```diff
@@ -19,8 +19,8 @@ const registrationFilters: FilterItem[] = [
   { field: ResultParam.Issn, i18nKey: 'registration.resource_type.issn' },
   { field: ResultParam.Doi, i18nKey: 'common.doi' },
   { field: ResultParam.Handle, i18nKey: 'registration.public_page.handle' },
-  // { field: ResultParam.GrantId, i18nKey: 'project.grant_id' }, // TODO: Uncomment when implemented in api-search
-  // { field: ResultParam.CourseCode, i18nKey: 'registration.resource_type.course_code' }, // TODO: Uncomment when implemented in api-search
+  { field: ResultParam.FundingIdentifier, i18nKey: 'registration.description.funding.funding_identifier' },
```

Comment:
Litt frista til å bruke oversettelsen som finnes til dette, da det skal være likt uansett hvor det er. Viktig at det man søker på her viser samme tittel som det som vises i wizard og landing page
Så skulle den kanskje ideelt sett vært i `common` da, men whatevz
```suggestion
  { field: ResultParam.FundingIdentifier, i18nKey: 'project.grant_id' },
```

Reply:
Debaterte litt med meg selv på denne. Burde kanskje "funding" vært en egen root i oversettelsefila?

Reply:
det er jo et eget konsept ja, så enig i at det kunne vært lurt 👍 
men kan godt bare gjenbruke den som finnes her nå da (selv om `grant_id` ikke er det beste navnet), for at det ikke skal eskalere ut til å rette opp mye andre plasser også